src/platform/web.rs

```diff
@@ -36,8 +36,13 @@ use crate::window::CustomCursor;
 use crate::window::{Window, WindowBuilder};
 use crate::SendSyncWrapper;
 
+#[cfg(wasm_platform)]
 use web_sys::HtmlCanvasElement;
 
+#[cfg(not(wasm_platform))]
+#[doc(hidden)]
+pub struct HtmlCanvasElement(());
```

Comment:
This was actually intentional, since I wanted to make sure it was unconstructable. But I'm fine with either, it's private anyway, and won't work even if someone compiled with `--cfg=docsrs`.

Reply:
I see, the only way I'm swinging more towards removing it is because I know I will forget why later and leaving a comment there seems overkill :sweat_smile:

But I'm fine with either.